ocaml/idl/markdown_backend.ml

```diff
@@ -130,7 +130,7 @@ let markdown_section_of_message printer x =
   printer (sprintf "%s (%s)"
     x.msg_name
     (String.concat ", "
-      ((if x.msg_session then "session_id s" else "")::
+      ((if x.msg_session then ["session ref session_id"] else []) @
```

Comment:
The problem was that, if `x.msg_session` is `false`, then the output was `", <args>"` rather than `"<args>"`. E.g. for `session.login_with_password`.